scripts/update_indices.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
I think the filename here isn't clear. Maybe something explicit like "download_support_files.sh"?